### DISCUSSION TITLE
Quick copy searched photos to clipboard

### DISCUSSION THREAD

**Discussion by Cookie755233** [johnlindquist/kit#1510] [Q&A]
Is it possible to create a script that grab the first few images and select the one I need, and directly saved into clipboard history (so that I don't have to open a new tab).

Ideally, inputing `img searchTerm` and it would show the options of the search results.

**Comment by johnlindquist** [comment]
Can you clarify what you mean by "grab the first few images"? Does that mean "first few selected images in Finder" or something else?

**Reply by Cookie755233** [reply]
I was expecting a list of few images of the search results if thats possible.

My current workflow often requires some images or icons, so I will google search, for example, "people", and copy image to clipboard for later use. 

Does it work with online search results? (the images weren't anywhere in finder)

**Reply by johnlindquist** [reply]
@Cookie755233 I'm working on Script Kit v3 and it will have a `await grid()` prompt made for searching/presenting images/etc.

Currently, you'd have to do something like how the "giphy" example works: https://github.com/johnlindquist/kit-examples/blob/main/scripts/giphy-search.js
